debian/control

```diff
@@ -0,0 +1,21 @@
+Source: libmirisdr5
+Section: hamradio
+Priority: optional
+Maintainer: ericek111 <erik.brocko@letemsvetemapplem.eu>
```

Comment:
I mainly meant here :D Pretty please, could you change it to `erik@brocko.eu`, too? Thank you

Reply:
Whoops :)